robottelo/cli/subscription.py

```diff
@@ -34,9 +35,11 @@ class Subscription(Base):
     def upload(cls, options=None):
         """Upload a subscription manifest."""
         cls.command_sub = 'upload'
+        timeout = 900 if bz_bug_is_open(1339696) else 300
```

Comment:
@sthirugn originally it was 5 minutes. you proposed to increase it to 10 minutes. then @rplevka provided update that after automation runs with all that data on satellite present it took him 11 mins to upload manifest. so 10 min may be not enough for last few manifest tests, that's why i increased it to 15 mins (900 secs) just in case